py_v_sdk/contract/atomic_swap_ctrt.py

```diff
@@ -13,8 +15,12 @@
 from py_v_sdk import data_entry as de
 from py_v_sdk import tx_req as tx
 from py_v_sdk import model as md
+from py_v_sdk import chain as ch
 from . import CtrtMeta, Ctrt
 
+from py_v_sdk.utils.crypto.hashes import sha256_hash
```

Comment:
```suggestion
from py_v_sdk.utils.crypto import hashes as hs
```
Then call `sha256_hash` like `hs.sha256_hash`